### DISCUSSION TITLE
Validation errors occurred. Please confirm the fields and submit it again.

### DISCUSSION THREAD

**Discussion by maowebsolutions** [kofimokome/cf7-message-filter#17] [Q&A]
I'm experiencing this problem when people are submitting the form the following message shows up and people who are legitimate sending the form is not seeing the thank you page. I tried and also got this message over and over again. 

"Validation errors occurred. Please confirm the fields and submit it again."

**Comment by kofimokome** [comment]
Hello @maowebsolutions , sorry for the late reply.
Are you still having the problem?
